ci/README.md

```diff
@@ -42,9 +42,11 @@ These issues effectively list PRs the merge bot has interacted with.
 
 To ensure security and a focused utility, the bot adheres to specific limitations:
 
-- The PR targets `master`, `staging`, or `staging-next`.
-- The PR only touches files located under `pkgs/by-name/*`.
-- The PR is authored by [@r-ryantm](https://nix-community.github.io/nixpkgs-update/r-ryantm/) or a [committer][@NixOS/nixpkgs-committers].
+- The PR targets one of the [development branches](#branch-classification).
+- The PR only touches packages located under `pkgs/by-name/*`.
+- The PR is either:
+  - authored by [@r-ryantm](https://nix-community.github.io/nixpkgs-update/r-ryantm/) or a [committer][@NixOS/nixpkgs-committers], or
+  - an automated backport created by label.
```

Comment:
It feels odd that the wording here is so different to the checklist item:

> an automated backport created by label.

> Backport by nixpkgs-ci.

Ideally, these would be more harmonised.

Reply:
Please have another look - is it better now?